src/libraries/System.IO.Ports/tests/SerialPort/GetPortNames.cs

```diff
@@ -18,7 +18,7 @@ public class GetPortNames : PortsTest
         /// <summary>
         /// Check that all ports either open correctly or fail with UnauthorizedAccessException (which implies they're already open)
         /// </summary>
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoNorServerCore))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34490", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
This issue is closed.
```suggestion
```

Reply:
I think you should submit a separate PR to see if this can be reenabled on Mono. This PR is not about System.IO.Ports so I would rather not change this and end up with two more rounds of CI in case this is still blocked on something.

> @jakobbotsch was the failure consistent? it may be flakyness in the Ports test, did you hit retry? in any case, the best is to file an issue to address the error.

I did not retry. I doubt this is just flakyness given the two separate failing tests and their log output (`File.Open` failed for `COM1` and `PortHelper.GetPorts` seems to correctly report the available ports, while `SerialPort.PortNames` does not).